.circleci/ecr_gc_docker/gc.py

```diff
@@ -146,6 +147,17 @@ def chunks(chunkable, n):
     for i in range(0, len(chunkable), n):
         yield chunkable[i : i + n]
 
+SHA_PATTERN = re.compile(r'\b[0-9a-f]{40}\b')
+def looks_like_git_sha(tag):
+    """Returns a boolean to check if a tag looks like a git sha
+
+    For reference a sha1 is 40 characters with only 0-9a-f and contains no
+    "-" characters
+    """
+    return all([
+        re.match(SHA_PATTERN, tag) is not None,
```

Comment:
Why don't you just do a full match (aka `^PATTERN$`) here?

Reply:
Updated to do full match